### PR TITLE
Add disable animation flag

### PR DESCRIPTION
**Pull request by DevarshRanpara**
Add disableanimation flag #22

### CODE REVIEW

File: lib/showcase.dart [hunk 1]
```diff
@@ -55,10 +57,10 @@ class Showcase extends StatefulWidget {
...
             "disposeOnTap is required if you're using onTargetClick"),
         assert(
-        disposeOnTap == null
-            ? true
-            : (onTargetClick == null ? false : true),
-        "onTargetClick is required if you're using disposeOnTap"),
+            disposeOnTap == null
+                ? true
+                : (onTargetClick == null ? false : true),
+            "onTargetClick is required if you're using disposeOnTap"),
```
Comment (simformsolutions) [inline lib/showcase.dart]: Not sure why this is changed

File: .gitignore [hunk 2]
```diff
@@ -58,3 +58,4 @@ build/
 instapk.log*
 ios/Frameworks/
 ios/Runner.xcworkspace/xcshareddata/
+example/ios/Flutter/flutter_export_environment.sh
```
Comment (simformsolutions) [inline .gitignore]: i think no need to update git ignore

Comment (DevarshRanpara) [reply .gitignore]: This file contains path information for resources required to project (Path can be very based on computer) which is not required in git.

File: lib/showcase.dart (hunk 1 above)
Comment (DevarshRanpara) [reply lib/showcase.dart]: We have to use both disposeOnTap and onTargetClick together,
That's why we have added assert for disposeOnTap as well.

Comment (simformsolutions) [reply lib/showcase.dart]: Is it due to code format.

Comment (DevarshRanpara) [reply lib/showcase.dart]: Is it due to code format.

**Pull request merged**